packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md

```diff
@@ -0,0 +1,54 @@
+
+- Context ID / envoy proxy
+
+- Pages new editing integration via HTTP render engine endpoint (e.g. editing/render)
+
+- XMCloud proxy personalization
+
+- Forms support
+
+This SPA is tailored to enhance development workflows and enable full utilization of XM Cloud’s capabilities, providing a seamless and efficient foundation for developers.
+
+## Components and Supporting Applications
+
+The following components and supporting applications have been added to the Angular base app to ensure compatibility with XM Cloud:
+
+- `XMCloud Angular`: Adds support for the Sitecore Context data, which simplifies connecting the application to XM Cloud and configuring the integration of multiple composable Sitecore products. Also includes example components for SXA projects.
+
+- `XMCloud Proxy`: Adds integration with XMCloud for the JSS SPA applications. Considered experimental.
+
+## Environment Variables
+
+The following environment variables can be set to configure the angular app. You can use the `.env` file located in the root of the app or set these directly in the environment (for example, in containers).
+
+| Parameter                              | Description                                                                                                                                |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `PROXY_HOST`                        | Your XM Cloud Proxy hostname is needed to build the app and execute the client-side requests against the proxy server. Default value `http://localhost:3000`                                                                                                                  |
+| `PROXY_BUILD_PATH`                              | Your XM Cloud Proxy server path is needed to build the app. The build output will be copied to the XMCloud Proxy application path. Default value `..\\node-xmcloud-proxy\\dist`.
+| `SITECORE_EDGE_CONTEXT_ID`                              | The Context ID, which covers many system configurations, required for connecting to the XM Cloud back end. This is an XM Cloud system environment variable. When the application runs on the XM Cloud rendering host, this value is always set to the preview Context ID.                   |
+| `SITECORE_API_KEY`                              | The API key for GRAPH_QL_ENDPOINT authentication. For Experience Edge, you can find the API key in the Sites dashboard by opening the actions menu for a site and navigating to Settings > Developer settings. Copy the value for SITECORE_API_KEY. For a preview endpoint (a CM instance either on XM Cloud or locally), use your XM Cloud preview API Key. When the application runs on the XM Cloud rendering host, this value is always set to your Sitecore Services Client API Key, and it will override any value set in your .env file.
```

Comment:
> For Experience Edge, you can find the API key in the Sites dashboard by opening the actions menu for a site and navigating to Settings > Developer settings.

This is true for JSS 21.5 and earlier but won't be applicable to Angular XMC release.

> When the application runs on the XM Cloud rendering host, this value is always set to your Sitecore Services Client API Key, and it will override any value set in your .env file.

I don't think API key will be used when app is deployed to XMC rendering host